File: Assignments/Week_03/generateEvenOdd.py
def genEven(n):
    for i in range(0,n):
        if(i % 2 == 0):
            print(i)

def genOdd(n):
    for i in range(0,n):
        if(i % 2 == 1):
            print(i)


def genEvenInRange(n):
    number = 0
    for i in range(0,n):
        if(number % 2 == 0):
            print (number)
        number += 2


def genOddInRange(n):
    number = 1
    for i in range(0,n):
        if(number % 2 == 1):
            print (number)
        number += 2


# genEven(25)
# genOdd(25)
# genEvenInRange(25)
genOddInRange(10)

File: Assignments/Week_03/generateEvenOdd.js
const genEven = (n) => {
    for (let i = 0; i< n; i++){
        if(i % 2 == 0){
            console.log(i)
        }
    }
}

const genOdd = (n) => {
    for (let i = 0; i< n; i++){
        if(i % 2 == 1){
            console.log(i)
        }
    }
}


const genEvenTillRange = (n) =>{
    let num = 0
    for(let i = 0; i < n; i++){
        if(num % 2 == 0){
            console.log(num)
        }
        num+=2
    }
}

const genOddTillRange = (n) =>{
    let num = 1
    for(let i = 0; i < n; i++){
        if(num % 2 == 1){
            console.log(num)
        }
        num += 2
    }
}

// genEven(15)
// genOdd(25)
// genEvenTillRange(10)
genOddTillRange(10)

